bin/gen-pgbouncer-conf.sh

```diff
@@ -134,22 +140,5 @@ EOFEOF
   let "n += 1"
 done
 
-# The password field for pgbouncer-stats was computed like so:
-#
-#   echo -n PASSWORDpgbouncer-stats | md5sum | awk '{print "md5" $1}'
-#
-# ...except PASSWORD was something else.  See ali/bin/launcher.sh for
-# the actual password.
-#
-# This is much like above where DB_MD5_PASS is calculated, and also
-# per https://pgbouncer.github.io/config.html, in the section
-# "Authentication file format".
-#
-# - jhw@prosperworks.com, 2016-02-09
-#
-cat >> /app/vendor/pgbouncer/users.txt << EOFEOF
-"pgbouncer-stats" "md572c5dd36a292efeaf13075b5eeb28693"
```

Comment:
This is a MD5 hash of a very low-value password which will stop working when this PR goes active in our stack anyhow.

I'm glad to see it go away but I'm not too paranoid about having it in our Git history or even in a history which might merge upstream.